src/meta/processors/usersMan/AuthenticationProcessor.cpp

```diff
@@ -74,19 +79,20 @@ void AlterUserProcessor::process(const cpp2::AlterUserReq& req) {
 
 void DropUserProcessor::process(const cpp2::DropUserReq& req) {
     folly::SharedMutex::WriteHolder wHolder(LockUtils::userLock());
-    auto ret = getUserId(req.get_account());
+    const auto& account = req.get_account();
+    auto ret = getUserId(account);
     if (!ret.ok()) {
-        if (req.get_missing_ok()) {
+        if (req.get_if_exists()) {
             handleErrorCode(cpp2::ErrorCode::SUCCEEDED);
```

Comment:
Please recheck "handleErrorCode",  IIRC "onFinished" has been called inside?

Reply:
> Please recheck "handleErrorCode", IIRC "onFinished" has been called inside?

emm... seems It's works well now,  "onFinished" have not call "handleErrorCode" inside.